Update braze-cohort-sync.md (#536)
amplitude/amplitude-docs: content/collections/destination-catalog/en/braze-cohort-sync.md

1. In Amplitude Data, click **Catalog** and select the **Destinations** tab.
2. In the Cohort section, click **Braze**.
3. Enter your App Group REST API key, select your endpoint, and paste your Data Import Key.

> 4. Select an Amplitude User Property and a Braze target object.

Reviewer: ```suggestion
4. Select an Amplitude user property and a Braze target object.
```

>   a. If you have selected "User alias" as the target object, ensure that you also populate the "User alias label" field.

Reviewer: ```suggestion
  * If you select **User alias** as the target object,  add a **User alias label**.
```

>   b. If you did not select "User alias" as the target object, you can leave the "User alias label" field empty.

Reviewer: ```suggestion
  * If you select a target object other than **User alias**, don't enter a **User alias label** .
```
